hs/app/reach/Main.hs

```diff
@@ -223,11 +225,18 @@ truthyEnv = \case
   Nothing -> False
   Just s -> not $ elem (map toLower s) [ "", "0", "false", "f", "#f", "no", "off", "n" ]
 
-mkVar :: IO Var
-mkVar = do
+mkVar :: FilePath -> IO Var
+mkVar cnf = do
   let packed = pure . pack
   let q e n = lookupEnv e >>= maybe n packed
   let m e n f = lookupEnv e >>= maybe n (\case "" -> n; j -> f j)
+  let def' p d = ifM (not <$> doesFileExist cnf) (pure d)
+        $ (filter (p' `L.isPrefixOf`) . lines <$> readFile cnf) >>= \case
+            [] -> pure d
+            cs -> pure $ case L.stripPrefix p' (last cs) of
+              Just c | length c > 0 -> c
+              _ -> d
+        where p' = p <> "="
```

Comment:
This is a hardcore little parser.

Reply:
Yeah... it's kinda inflexible too, and I'm not sure if that's a good thing or not.

I had it in the back of my head that we may want to make it smarter later but this works in the minimal case.

Reply:
As you know I like minimal things :) What you have makes the file "look like" something that could be `source`'d. That's cool. I think it is also feasible that some library we already use (yaml?) can serialize an arbitrary map and then we just load it. I bet (?) that would be less code and because it's someone else's problem, the parser is probably robustier